packages/http-client-js/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@typespec/http-client-js",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "main": "dist/src/index.js",
+  "scripts": {
+    "build-src": "babel src -d dist/src --extensions .ts,.tsx",
+    "build": "tsc -p . && npm run build-src",
+    "clean": "rimraf ./dist",
+    "lint": "eslint . --max-warnings=0",
+    "lint:fix": "eslint . --fix",
+    "watch-src": "babel src -d dist/src --extensions .ts,.tsx --watch",
+    "watch-tsc": "tsc -p . --watch",
+    "watch": "concurrently --kill-others \"npm run watch-tsc\" \"npm run watch-src\"",
+    "test:scenarios": "vitest run scenarios.test.ts",
+    "test": "npm run test:scenarios",
+    "build-todo": "rimraf sample/output/todo dist/sample/output/todo && tsp compile sample/main.tsp --emit @typespec/http-client-js --output-dir sample/output/todo && babel sample/output/todo -d dist/sample/output/todo --extensions .ts,.tsx && npx prettier sample/output --write",
+    "format": "prettier . --write",
+    "build:test": "tsc -p tsconfig.test.json",
+    "start:server": "npx tsp-spector server start node_modules/@typespec/http-specs/specs --coverageFile ./temp/tsp-spector-coverage-javascript-standard.json --debug",
+    "stop:server": "npx tsp-spector server stop",
+    "test:e2e": "npm run emit:e2e && node eng/scripts/run-e2e-tests.js",
+    "run:e2e": "node eng/scripts/run-e2e-tests.js",
+    "emit:e2e": "node eng/scripts/emit-e2e.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "peerDependencies": {
+    "@typespec/compiler": "workspace:~",
+    "@typespec/emitter-framework": "workspace:~",
+    "@typespec/http": "workspace:~",
+    "@typespec/http-client": "workspace:~",
+    "@typespec/rest": "workspace:~"
+  },
+  "dependencies": {
+    "@alloy-js/core": "^0.5.0",
+    "@alloy-js/typescript": "^0.5.0",
+    "prettier": "~3.4.2"
+  },
+  "devDependencies": {
+    "@alloy-js/babel-preset": "^0.1.1",
+    "@babel/cli": "^7.24.8",
+    "@babel/core": "^7.26.0",
+    "@rollup/plugin-babel": "^6.0.4",
+    "@types/yargs": "~17.0.33",
+    "@typespec/http-specs": "0.1.0-alpha.9",
+    "@typespec/spector": "workspace:~",
+    "@typespec/ts-http-runtime": "0.1.0",
+    "@typespec/versioning": "workspace:~",
+    "@vitest/ui": "^3.0.3",
+    "chalk": "^2.4.2",
```

Comment:
nit: we have been using picocolors in the other packages

Reply:
moved to picocolors